packages/communication/src/commons.ts

```diff
@@ -0,0 +1,70 @@
+import { PromiEvent, Transaction, TransactionConfig, TransactionReceipt } from 'web3-core'
```

Comment:
Looks like these are all types. Can we name this `types.ts`?

Reply:
👍